win32_event_log/datadog_checks/win32_event_log/data/conf.yaml.example

```diff
@@ -44,6 +44,7 @@ init_config:
     # service: <SERVICE>
 
 ## Every instance is scheduled independent of the others.
+## Everything under this section is sent as events.
```

Comment:
This needs to be reflected in /assets/configuration/spec.yaml too

Reply:
@ChristineTChen having trouble finding where a description for a `template` level object should go...

Reply:
So you'd need to modify this description: https://github.com/DataDog/integrations-core/blob/master/win32_event_log/assets/configuration/spec.yaml#L46
```
  - template: instances
     overrides:
       description: |
         Every instance is scheduled independent of the others.
         Everything under this section is sent as events.
```


instances general template: https://github.com/DataDog/integrations-core/blob/master/datadog_checks_dev/datadog_checks/dev/tooling/templates/configuration/instances.yaml#L3-L4